packages/dds/map/src/mapKernel.ts

```diff
@@ -7,7 +7,7 @@ import type { TypedEventEmitter } from "@fluid-internal/client-utils";
 import type { IFluidHandle } from "@fluidframework/core-interfaces";
 import { assert, unreachableCase } from "@fluidframework/core-utils/internal";
 import type { IFluidSerializer } from "@fluidframework/shared-object-base/internal";
-import { ValueType, bindHandles } from "@fluidframework/shared-object-base/internal";
```

Comment:
You can also delete this function, it's not used anywhere else, and isn't an ideal implementation.  @Abe27342 what's the task you pinged me about fixing the function? We can close it if this PR works out.